tools/scripts/update-bumpenvs-yaml.py

```diff
@@ -33,26 +33,30 @@
 PROJECT = "environments"
 BASE_URL = f"https://circleci.com/api/v1.1/project/github/{USER}/{PROJECT}"
 
+JOB_SUFFIXES = [
+    "tf1-cpu",
+    "tf2-cpu",
+    "tf25-cpu",
+    "tf26-cpu",
+    "tf1-gpu",
+    "tf2-gpu",
+    "tf25-gpu",
+    "tf26-gpu",
+]
+
 EXPECT_JOBS = {
```

Comment:
I feel like all of these values should really just be lists? Eh.

Reply:
I think we're using it for equality checks here and there. this is an admin script, python performance doesn't really matter.

Reply:
Just semantically, I mean, not for performance or anything. Doesn't matter either way.